src/Microsoft.OpenApi.Hidi/Formatters/PowerShellFormatter.cs

```diff
@@ -350,12 +350,12 @@ private static JsonSchema CopySchema(JsonSchema schema, JsonSchema newSchema)
             }
             if (schema.GetDiscriminator() == null && newSchema.GetDiscriminator() is { } discriminator)
             {
-                schemaBuilder.Discriminator(discriminator.PropertyName, discriminator.Mapping, discriminator.Extensions);
-            }
-            if (schema.GetOpenApiExternalDocs() == null && newSchema.GetOpenApiExternalDocs() is { } externalDocs)
-            {
-                schemaBuilder.OpenApiExternalDocs(externalDocs);
+                schemaBuilder.Discriminator(discriminator.PropertyName,discriminator.Mapping, discriminator.Extensions);
             }
+            //if (schema.GetExternalDocs() == null && newSchema.GetExternalDocs() is { } externalDocs)
+            //{
+            //    schemaBuilder.ExternalDocs(new Uri(externalDocs), externalDocs.Description);
+            //}
```

Comment:
@darrelmiller why did we get rid of ExternalDocs here?

Reply:
I don't know.  This seems like a mistake to me.